<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Disa PC File Manager</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTreeView" name="treeView">
         <attribute name="headerDefaultSectionSize">
          <number>100</number>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>521</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>181</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btnDelete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>350</y>
            <width>181</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
    background-color: #ff9c85;
	border-style: solid;
}

QPushButton:!hover
{
     background-color: #ff522a;
}

QPushButton:!enabled
{
     background-color: #f8cdc3;
}</string>
          </property>
          <property name="text">
           <string>Delete selected</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnAbout">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>250</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>About</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>80</y>
            <width>181</width>
            <height>161</height>
           </rect>
          </property>
          <property name="title">
           <string>Filters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="0">
            <widget class="QCheckBox" name="checkBox_OGG">
             <property name="text">
              <string>*.OGG</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="checkBox_PNG">
             <property name="text">
              <string>*.PNG</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox_DAT">
             <property name="text">
              <string>*.DAT</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_ARC">
             <property name="text">
              <string>*.ARC</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_MPD">
             <property name="text">
              <string>*.MPD</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkBox_FFM">
             <property name="text">
              <string>*.FFM</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_FAD">
             <property name="text">
              <string>*.FAD</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="checkBox_TX2">
             <property name="text">
              <string>*.TX2</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_MPP">
             <property name="text">
              <string>*.MPP</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="checkBox_PBD">
             <property name="text">
              <string>*.PBD</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="checkBox_PHD">
             <property name="text">
              <string>*.PHD</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="checkBox_GEO">
             <property name="text">
              <string>*.GEO</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="btnOpen">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>181</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnExtract">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>280</y>
            <width>181</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Extract selected</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnInsert">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>310</y>
            <width>181</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Insert files</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSave">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
         <widget class="ItemPreview" name="imagePreview">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>380</y>
            <width>181</width>
            <height>181</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: gray;</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSaveAs">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>30</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Save As</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnAboutQt">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>250</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>About Qt</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnExtractImage">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>570</y>
            <width>181</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Export image</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ItemPreview</class>
   <extends>QGraphicsView</extends>
   <header>ItemPreview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
